theme/templates/base.html

```diff
@@ -1,10 +1,19 @@
 <!DOCTYPE html>
-<html lang="{{ DEFAULT_LANG }}">
+<html lang="{{ DEFAULT_LANG }}" prefix="og: http://ogp.me/ns#">
 <head>
   {% block head %}
     <title>{{ SITENAME }} - {% block title %}{% endblock title %}</title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    {% if page is defined %}
+    {% if page.og_url %}<meta property="og:url" content="{{ page.og_url}}" />{% endif %}
```

Comment:
Le 23/06/2016 à 13:57, Rémy HUBSCHER a écrit :
>
> In theme/templates/base.html 
> <https://github.com/AFPy/pyconfr_2016/pull/21#discussion_r68221484>:
>
> >  <head>
> >    {% block head %}
> >      <title>{{ SITENAME }} - {% block title %}{% endblock title %}</title>
> >      <meta charset="utf-8" />
> >      <meta name="viewport" content="width=device-width, initial-scale=1.0">
> > +
> > +    {% if page is defined %}
> > +    {% if page.og_url %}<meta property="og:url" content="{{ page.og_url}}" />{% endif %}
>
> Oh so it will be configured only for this page right?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/AFPy/pyconfr_2016/pull/21/files/f2878f6fcc1a805562b2eb40f894482b10d118c9#r68221484>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ABGf9wNe3ybLRwajv0535nvJFfYYSQQxks5qOnS3gaJpZM4I8Gas>.
>
Yes